webgrid/tests/client_helpers.py

```diff
@@ -0,0 +1,108 @@
+"""
```

Comment:
I'd like to see this module at `webgrid.testing` instead of `webrid.tests.client_helpers`.  That aligns with what we are doing in other libraries like Keg and KegElements.

Reply:
Done, thanks for naming tip.